public/js/charts/worstPMGraph.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
Why?

Reply:
@kilemensi Sorry totally forgot about this....had a console.log in there. I'll delete it